doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -932,6 +932,8 @@ Modem libraries
 
 * :ref:`lte_lc_readme` library:
 
+  * Added the :kconfig:option:`CONFIG_LTE_PLMN_SELECTION_OPTIMIZATION` to enable PLMN selection optimization on network search.
+    This options is enabled by default. For more information see the ``AT%FEACONF`` command documentation in the `nRF91x1 AT Commands Reference Guide`_.
```

Comment:
```suggestion
  * Added the :kconfig:option:`CONFIG_LTE_PLMN_SELECTION_OPTIMIZATION` to enable PLMN selection optimization on network search.
    This option is enabled by default. 
    For more information, see the ``AT%FEACONF`` command in the `nRF91x1 AT Commands Reference Guide`_.
```
Please add a comment on PR https://github.com/nrfconnect/sdk-nrf/pull/15961 for this entry to include this in the 2.7 release note, and once added you could remove the entry from this PR. 
cc: @b-gent

Reply:
OK